openmetadata-ui/src/main/resources/ui/src/locale/languages/zh-cn.json

```diff
@@ -631,7 +631,11 @@
     "interval-unit": "间隔单位",
     "invalid-condition": "无效条件",
     "invalid-name": "无效名称",
+    "is": "Is",
```

Comment:
can we use openAI to translate some of these instead of leaving them in english?